crates/typst-syntax/src/parser.rs

```diff
@@ -1509,120 +1489,139 @@ fn pattern_leaf<'s>(
     }
 }
 
-/// Manages parsing of a stream of tokens.
+/// Manages parsing a stream of tokens into a tree of SyntaxNodes.
```

Comment:
General note: Please wrap comments at 80 rather than 90 columns.

Reply:
:+1: Updated my comment wrapping extension to wrap at 80. Will fix these.